.github/workflows/lint-test.yml

```
@@ -151,6 +151,16 @@ jobs:
         pip install -U build
         python -m build
 
+    # run the libcst parsers and check for errors
+    - name: libcst codemod
+      run: |
+        nox -s codemod
```

```suggestion
        nox -s codemod -- run-all
```
just to be explicit, and not rely on `session.interactive`

---

https://github.com/DisnakeDev/disnake/pull/708/commits/81dd94e09ab2c49fc8d7dad7321f69718f8971ac